moml/core/__init__.py

```
@@ -29,30 +29,76 @@
+    from .molecular_graph_processor import (
+        MolecularGraphProcessor,
+        create_graph_processor,
+        mol_file_to_graph,
+        create_molecular_graph_json,
+        batch_create_graphs_from_molecules,
+        collate_graphs,
+        graph_to_device,
+        find_charges_file,
+        read_charges_from_file,
+    )
+except ImportError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)